samtranslator/internal/schema_source/aws_serverless_api.py

```diff
@@ -158,6 +158,7 @@ class EndpointConfiguration(BaseModel):
 
 Name = Optional[PassThroughProp]
 DefinitionUriType = Optional[Union[str, DefinitionUri]]
+MergeDefinitions = Optional[SamIntrinsicable[bool]]
```

Comment:
Why `SamIntrinsicable`? If it's a property only relevant to SAM, we shouldn't deal with intrinsics as `AWS::LanguageExtensions` takes care of them before SAM. Applies to all types of `MergeDefinitions` in this PR.

Reply:
How do you guarantee that the template also uses `AWS::LanguageExtensions`?

Reply:
We ask that customers that encounter an issue add it: https://github.com/aws/serverless-application-model/issues/2533

`AWS::LanguageExtensions` resolves intrinsics generically and more thoroughly than the unsustainable ad-hoc approach SAM had so far.